efficientdet/anchors.py

```diff
@@ -350,25 +352,24 @@ def _generate_detections_tf(cls_outputs,
   scores = tf.math.sigmoid(cls_outputs)
   # apply bounding box regression to anchors
   boxes = decode_box_outputs_tf(box_outputs, anchor_boxes)
-
-  top_detection_idx, scores = tf.image.non_max_suppression_with_scores(
-      boxes,
-      scores,
-      max_boxes_to_draw,
+  top_detection_idx, scores, _ = tf.raw_ops.NonMaxSuppressionV5(
+      boxes=boxes,
+      scores=scores,
+      max_output_size=max_boxes_to_draw,
       iou_threshold=iou_threshold,
       score_threshold=min_score_thresh,
       soft_nms_sigma=soft_nms_sigma)
   boxes = tf.gather(boxes, top_detection_idx)
 
   image_size = utils.parse_image_size(image_size)
   detections = tf.stack([
-      tf.cast(tf.tile(image_id, tf.shape(top_detection_idx)), tf.float32),
+      tf.cast(tf.tile(image_id, tf.shape(top_detection_idx)), dtype),
```

Comment:
I remember this change causes some unittests failed. Could run those unittests?:

Reply:
I add it later.